src/libraries/System.Data.Common/tests/TrimmingTests/CreateSqlXmlReader.cs

```diff
@@ -0,0 +1,13 @@
+using System.Data.SqlTypes;
```

Comment:
Have we been putting a copyright on these tests? We probably should - there is a copyright on all the tests in the repo.

Reply:
No 😬 - will start now.